gneiss/regression/_ols.py

```diff
@@ -298,3 +318,144 @@ def r2(self):
 
         sst = sse + ssr
         return 1 - sse / sst
+
+    @property
+    def mse(self):
+        sse = sum([r.ssr for r in self.results])
+        dfe = self.results[0].df_resid
+        return sse / dfe
+
+    def loo(self, regularized=False, **kwargs):
+        """ Leave one out cross-validation.
+
+        Calculates summary statistics for each iteraction of
+        leave one out cross-validation, specially `mse` on entire model
+        and `pred_err` to measure prediction error.
+
+        Parameters
+        ----------
+        regularized : bool
+            Specifies if a regularization procedure should be used
+            when performing the fit.
+        **kwargs : dict
+           Keyword arguments used to tune the parameter estimation.
+
+        Returns
+        -------
+        pd.DataFrame
+           mse : np.array, float
+               Mean sum of squares error for each iteration of
+               the cross validation.
+           pred_err : np.array, float
+               Prediction mean sum of squares error for each iteration of
+               the cross validation.
+
+        See Also
+        --------
+        fit
+        """
+        params = {'fit_regularized': False}
+        for key in params:
```

Comment:
The main reason for having the kwargs argument was to pass these into the `fit` function, which have a ton of parameters in them.  One advantage of doing this is if the `fit` function in statsmodels changes, we aren't forced to change our APIs to fit their changes.